Sources/Xcodeproj/pbxproj().swift

```diff
@@ -426,7 +426,7 @@ func xcodeProject(
         // .xcconfig, if we have one.  This lets it override project settings.
         targetSettings.xcconfigFileRef = xcconfigOverridesFileRef
 
-        targetSettings.common.TARGET_NAME = target.name
+        targetSettings.common.TARGET_NAME = target.c99name
```

Comment:
Hmm, what was the problem again? I am able to build and test a package with non-c99 target:

$ mkdir "foo bar"
$ cd "foo bar"
$ swift package init
$ swift package generate-xcodeproj
$ xcodebuild test -scheme "foo bar-Package"

Reply:
Ah the comment might not be useful. This happened for generated projects where the executable contained
$ mkdir "foo-bar"
$ cd "foo-bar"
$ swift package init --type executable
$ swift package generate-xcodeproj
$ xcodebuild test -scheme "foo bar-Package"